esmvaltool/recipes/recipe_autoassess_landsurface_surfrad.yml

```diff
@@ -0,0 +1,75 @@
+# ESMValTool
+# recipe_autoassess_landsurface_surfrad.yml
+---
+documentation:
+  description: |
+    Recipe that runs the Autoassess Land-surface assessment area diagnostic.
+    Land-sea masks are not averaged over time (annual) but are fixed, i.e.
+    using sftlf (lat-lon) grid. The original  Autoassess diagnostic used
+    time-dependent .pp masks these can still be implemented if present in
+    autoassess_source. The recipe is fully working but masks need to be
+    checked.
+
+  authors:
+    - predoi_valeriu
+
+  maintainer:
+    - predoi_valeriu
+
+datasets:
+  - {dataset: MPI-ESM-LR,  project: CMIP5,  exp: amip,  ensemble: r1i1p1,  start_year: 1992,  end_year: 2002}
+  - {dataset: MPI-ESM-MR,  project: CMIP5,  exp: amip,  ensemble: r1i1p1,  start_year: 1992,  end_year: 2002}
+  - {dataset: inmcm4,      project: CMIP5,  exp: amip,  ensemble: r1i1p1,  start_year: 1992,  end_year: 2002}
+
+preprocessors:
+  pp_aa_area:
+    regrid:  # NOT USED
+      target_grid: 0.15x0.15
+      scheme: linear
+
+diagnostics:
+  aa_landsurf_surfrad:
+    description: Autoassess test diag for Land-Surface Surfrad.
+    variables:
+      rsns: # Surf SW net all sky
+        derive: true
+        force_derivation: false
+        mip: Amon
+        additional_datasets:
+          - {dataset: CERES-EBAF,  project: obs4mips,  level: L3B,  version: Ed2-7,  start_year: 2001,  end_year: 2012, tier: 1}
+      sftlf:
+        mip: fx
+      rlns: # Surf LW net all sky
```

Comment:
Similarly, at least for CMIP6 this should be rls instead of rlns.

Reply:
similarily too, this is [derived](https://github.com/ESMValGroup/ESMValCore/blob/master/esmvalcore/preprocessor/_derive/rlns.py) and it's got its own custom [table](https://github.com/ESMValGroup/ESMValCore/blob/master/esmvalcore/cmor/tables/custom/CMOR_rlns.dat) - I think we discussed these back in the day. Do you think these two need to change?

Reply:
As above, the derived variable rnls is fine and is the same thing as the CMIP6 variable rls.

Reply:
OK excellent! :beer: